src/computability/language.lean

```diff
@@ -54,14 +54,17 @@ lemma mul_def (l m : language α) : l * m = image2 (++) l m := rfl
 
 /-- The star of a language `L` is the set of all strings which can be written by concatenating
   strings from `L`. -/
+  
+@[simp] lemma mem_add (l m : language α) (x : list α) : x ∈ l + m ↔ x ∈ l ∨ x ∈ m := iff.rfl
```

Comment:
The correct thing to do here is to remove the `simp` attribute from `mem_add`. No need to reorder.